src/app/app.module.ts

```diff
@@ -1,18 +1,28 @@
 import {BrowserModule} from '@angular/platform-browser';
 import {NgModule} from '@angular/core';
-import {FormsModule} from '@angular/forms';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 import {HttpModule} from '@angular/http';
 
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

Comment:
In Angular 4 you need either `BrowserAnimationsModule` or `BrowserNoopAnimationsModule`